src/main/java/io/github/org/programming/bot/commands/info/UserInfoCommand.java

```diff
@@ -86,17 +87,17 @@ private void getInfo(@NotNull Member member, @NotNull User user,
             AVATAR = "No Avatar";
         }
 
-        EmbedBuilder em = new EmbedBuilder().setColor(Color.GREEN);
+        EmbedBuilder em = new EmbedBuilder().setColor(Color.GRAY);
         em.setDescription("🕵️   **User information for " + member.getUser().getName() + ":**")
-            .addField("Name / Nickname", NAME, false)
-            .addField("User Tag", TAG, false)
-            .addField("ID", ID, false)
-            .addField("Current Status", STATUS, false)
-            .addField("Current Activities", String.valueOf(GAMES), false)
-            .addField("Roles", String.valueOf(ROLES), false)
-            .addField("Guild Joined", GUILD_JOIN_DATE, false)
-            .addField("Discord Joined", DISCORD_JOINED_DATE, false)
-            .addField("Avatar-URL", AVATAR, false);
+            .addField("Name / Nickname", NAME, true)
+            .addField("User Tag", TAG, true)
+            .addField("ID", ID, true)
+            .addField("Current Status", STATUS, true)
+            .addField("Current Activities", String.valueOf(GAMES), true)
+            .addField("Roles", String.valueOf(ROLES), true)
+            .addField("Guild Joined", GUILD_JOIN_DATE, true)
+            .addField("Discord Joined", DISCORD_JOINED_DATE, true)
+            .addField("Avatar-URL", AVATAR, true);
```

Comment:
what is this boolean ?

Reply:
Informs you weather the field should be inlined like this
<img width="748" alt="Screenshot 2022-09-11 at 19 45 24" src="https://user-images.githubusercontent.com/67903886/189543997-55cc2599-d48a-45bc-8b2b-39ed579b8c84.png">